.github/workflows/validate-pr.yml

```
@@ -42,15 +42,23 @@ jobs:
           # Strip empty lines
           desc=$(echo "$block" |  awk NF)
 
-          if [ -z "$desc" ]; then
-            echo "::error::Changelog section is empty. Please provide a description for the changelog."
-            exit 1
-          fi
+          if [ "$HAS_IMPACT_LABEL" = "true" ]; then
+            if [ -z "$desc" ]]; then
```

```suggestion
            if [ -z "$desc" ]; then
```

Can't tell super well, but that's an extra `]` right?

---

Thanks for spotting!